source/network/ODSocketServer.cpp

```diff
@@ -52,7 +51,7 @@ bool ODSocketServer::createServer(int listeningPort)
     }
 
     mSockSelector.add(mSockListener);
-
+    mNewClient = new ODSocketClient;
```

Comment:
That was just pure random luck, I wanted to look into making the server try an extra port if it failed. (As it seems, at least on my system there is some delay before they are released after the game is exited, which can be a bit frustrating when testing things.) That seemed to require a lot more code change, so I left it for now, but I noticed that there was a leak there.